version.json

```diff
@@ -1,14 +1,4 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "17.4",
-  "assemblyVersion": "17.4",
-  "buildNumberOffset": 0,
-  "cloudBuild": {
-    "buildNumber": {
-      "enabled": true,
-      "includeCommitId": {
-        "when": "never"
-      }
-    }
-  }
```

Comment:
So, not using the cloudBuild mechanism outline here anymore: https://github.com/dotnet/Nerdbank.GitVersioning/blob/main/doc/cloudbuild.md#automatically-match-cloud-build-numbers-to-to-your-git-version
Also, `assemblyVersion` wasn't needed since it isn't different than `version`. And `buildNumberOffset` is deprecated, and wasn't needed anyway.